app/instance-initializers/violations-helper.js

```diff
@@ -0,0 +1,11 @@
+import ViolationsHelper from 'ember-a11y-testing/utils/violations-helper';
+
+export function initialize(application) {
+  window.violationsHelper = new ViolationsHelper();
```

Comment:
No serious performance hits, but exposing globals is something you want to be very sure about :)